src/modules/integrator-guide/pages/compose-configuration/page-blocks.adoc

```diff
@@ -743,3 +743,31 @@ Defines the report the page bock will use when showing data.
 Defines the display element from the selected report the page block will use when showing data.
 
 |===
+
+[#page-block-geometry]
+== Geometry
+
+The *geometry page block* allows you to show a map with a marker, lines and polygons from different geometry values of different record sources which are rendered on the map.
```

Comment:
`...record sources which are rendered on the map.` -> `...record sources.`

The being rendered on the map is already explained in the first bit -- `...allows you to show a map with...`